params.yaml

```diff
@@ -0,0 +1,41 @@
+graph-defaults:
+  levels: 4
+  hierarchical: true
+  name: hierarchical
+
+baseline-defaults: &baseline-defaults  # defaults across training and evaluation stage
```

Comment:
these `&` and `<<: *` look cool. Could you add some comments to explain what the different uses mean? Like it isn't super clear what `num_nodes: &num_nodes 2` vs `nodes: *num_nodes` vs `num_nodes: 1` means

Reply:
I would also add a comment here that these will be passed as CLI args to `python -m neural_lam.train_model`